src/main/java/rs/hybridit/repository/BookCopyRepository.java

```diff
@@ -0,0 +1,8 @@
+package rs.hybridit.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import rs.hybridit.model.BookCopy;
+
+public interface BookCopyRepository extends JpaRepository<BookCopy, Long> {
```

Comment:
you missed to add @Repository but it works :)
you don't need to annotate repository class, spring knows, so remove from other repositories